src/tasks/models/tasksManager.ts

```diff
@@ -93,6 +97,16 @@ export class TasksManager {
     await this.updateJobPercentage(job.id, calculatedPercentage);
   }
 
+  private async handleSeedingTask(job: IJobResponse<unknown, unknown>, task: ITaskResponse<unknown>): Promise<void> {
+    this.logger.debug({ msg: `Updating seeding job percentage; ${task.type}` });
```

Comment:
remove irrelevant log, this log appeared inside `this.updateJobPecentage` function